src/jobsautoreport/query.py

```diff
@@ -35,21 +37,30 @@ def _get_query_all_jobs(from_date: datetime, to_date: datetime) -> dict:
             }
         }
 
-    def query_all_jobs(
+    def query_jobs(self, from_date: datetime, to_date: datetime) -> list[JobDetails]:
+        query = self._get_query_all_objects(from_date, to_date)
+        return self._query_jobs_and_log(query)
+
+    def query_step_events(
         self, from_date: datetime, to_date: datetime
-    ) -> list[JobDetails]:
-        query = self._get_query_all_jobs(from_date, to_date)
-        return self._query_and_log(query)
+    ) -> list[StepEvent]:
+        query = self._get_query_all_objects(from_date, to_date)
+        return self._query_step_events_and_log(query)
+
+    def _query_jobs_and_log(self, query: dict) -> list[JobDetails]:
+        logger.debug("OpenSearch query: %s", query)
+        objects = self._scan(query, self._jobs_index)
+        return utils.parse_jobs(objects)
 
-    def _query_and_log(self, query: dict) -> list[JobDetails]:
+    def _query_step_events_and_log(self, query: dict) -> list[StepEvent]:
         logger.debug("OpenSearch query: %s", query)
-        jobs = self._scan(query)
-        return utils.parse_jobs(jobs)
+        objects = self._scan(query, self._steps_index)
+        return utils.parse_step_events(objects)
```

Comment:
maybe we can filter here directly on the steps matching `packet-setup`, as there are much more steps than jobs, and we are really interested into one part of the data